salt/haproxy/init.sls

```diff
@@ -62,8 +62,10 @@ haproxy-restart:
     - timeout: 60
     - onchanges:
       - file: /etc/caasp/haproxy/haproxy.cfg
+{% if not salt.caasp_cri.needs_docker() %}
     - require:
       - service: container-feeder
```

Comment:
Why is the `container-feeder` service not required for docker?

Reply:
Our images have docker enabled by default. On the 1st boot container-feeder will automatically import all the pre-built images into docker. That doesn't happen with cri-o. Hence we have to ensure the container-feeder sls file has been applied when docker is not being used.